src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
```

Comment:
for that interface will be better to create a new file and export him to App component for your state, and here

Reply:
agree